### ISSUE TITLE
Damaged tools have no durability and no model.

### ISSUE THREAD

**Issue by Tokeli** [Randores/Randores2#1]
Forge 14.22.1.2479
Randores2-2.0.0-Pre2

When any Randores tools take damage, their model turns into a blank purple cube, and their durability vanishes. I'm unsure as to whether this means they have infinite durability now as I haven't been able to sit around long enough to test them!

**Comment by SocraticPhoenix** [comment]
I just tested this on forge 14.22.0.2459 and I couldn't reproduce the bug. I can't download the new forge version right now, but it would be helpful if you could either test on the above version of forge, or check if there are any crash reports or errors in the logs...

**Comment by Tokeli** [comment]
I did some more testing and realized the issue only happens on SMP. I setup a 2459 server locally with nothing but Randores and JEI, and can confirm that it does happen there (as well as on 2481, because that's what I was actually using in the first place, accidentally wrote the wrong version!)

Tools function perfectly fine in singleplayer, but switch over to a server and they will lose their models when they take damage.

Notably, I got a sledgehammer that didn't have any durability in singleplayer as well, but it retained its model. All the other tools seem to work fine. No errors in any of the logs however.
